lib/core/macros.nim

```diff
@@ -1626,9 +1626,18 @@ macro unpackVarargs*(callee: untyped; args: varargs[untyped]): untyped =
     result.add args[i]
 
 proc getProjectPath*(): string = discard
-  ## Returns the path to the currently compiling project, not to
-  ## be confused with ``system.currentSourcePath`` which returns
-  ## the path of the current module.
+  ## Returns the path to the currently compiling project,
+  ## the path of the Nim file being compiled,
+  ## not to be confused with ``system.currentSourcePath`` which returns
+  ## the path of the current module. Its available at compile-time.
```

Comment:
Firstly, its -> it's or it is (this probably applies throughout the PR)

Also, if I have a file in `/home/jjp/code/proj/some_file.nim`, it seems that `getProjectPath` returns `/home/jjp/code/proj` while `system.currentSourcePath` returns `/home/jjp/code/proj/some_file.nim`, but the phrasing "the path of the Nim file being compiled" implies the latter. Maybe incorporating the word "directory" would be better. "project" lightly implies "directory" but I think making it explicit would be better anyway.

And regarding:
> Its available at compile-time

Based on some testing I did, I think this proc is practically *only* available at compile-time because otherwise I just get an empty string, so you need to add the word "only".